Macros/Read_address_points.sas

```diff
@@ -369,8 +373,10 @@
 		stantoncommons = "Stanton Commons (2018)"
 		cluster2017 = "Neighborhood Clusters (2017)"
 		ZIPCODE4 = "Zip +4"
-		XCOORD = "X Coordinate"
-		YCOORD = "Y Coordinate"
+		XCOORD = "X coordinate of address point (MD State Plane Coord., NAD 1983 meters)"
+		YCOORD = "Y coordinate of address point (MD State Plane Coord., NAD 1983 meters)"
```

Comment:
@ptatian the macro now implements that change. It's kind of quirky because there is a rename statement in this data step, so even though the label statement looks like it's labeling the XCOORD and YCOORD variables, the rename statement is already changing them to X and Y.